packages/orderbook/test/js_api/gui.test.ts

```
@@ -503,7 +533,7 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Gui', async function ()
 	});
 
 	describe('deposit tests', async () => {
-		let gui: DotrainOrderGui;
+		let gui = new DotrainOrderGui();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'gui' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)